updater/reports/ReportGitDownload.py

```diff
@@ -12,10 +12,10 @@ def updateDailyData(self):
 		if len(self.data) == 0:
 			self.header = [
 				"date",
-				"clones/day",
-				"fetches/day",
-				"clone traffic/day [GB]",
-				"fetch traffic/day [GB]",
+				"clones",
+				"fetches",
+				"clone traffic [GB]",
+				"fetch traffic [GB]",
```

Comment:
The header is only rewritten if the `self.data` is empty. Since we reference the new header names in the JS code, the charts are empty 😢 

I wonder what's the best approach here. My gut feeling is to rewrite the header here. What do you think?